CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [1.0.0] - 2023-01-23
+### Changed
+- `ProviderFilters.CustomerSegments` is now an array.
```

Comment:
I think the process is to add the changelog only once the new version is published, so you have to do it in a following PR. Also, when you do it, can you please add a link to the official changelong for context?

Reply:
Sure, I'll rollback these changes for now.